src/compiler/style.ts

```diff
@@ -1,10 +1,8 @@
-'use strict';
+import * as vlEncDef from '../encdef';
 
-require('../globals');
+import {X, Y, ROW, COL} from '../consts';
 
-var vlEncDef = require('../encdef');
-
-module.exports = function(encoding, stats) {
+export function def(encoding, stats) {
```

Comment:
style doesn't produce a part of vega and thus shouldn't be named `def`.

I don't know how to just export in a way that compiler.ts can call `vlstyle(...)` rather than `vlstyle.def(...)`